contracts/token/RarePizzasBox.sol

```diff
@@ -114,68 +127,61 @@ contract RarePizzasBox is
         override(ERC721Upgradeable, IERC721MetadataUpgradeable)
         returns (string memory)
     {
-        require(_exists(tokenId), 'RAREPIZZA: URI query for nonexistant token');
+        require(_exists(tokenId), 'RAREPIZZA: does not exist yet, paisano');
         return string(abi.encodePacked(_uriBase, getUriString(_tokenBoxArtworkURIs[tokenId])));
     }
 
-    // Member Functions
-
-    /**
-     * Get the current bitcoin price in wei
-     */
-    function getBitcoinPriceInWei() public view returns (int256) {
-        return bitcoinPriceInWei;
-    }
+    // IRarePizzasBoxAdmin
 
-    /**
-     * allows the contract owner to mint up to a specific number of boxes
-     */
-    function mint(address to, uint8 count) public virtual onlyOwner {
-        require(count > 0, 'RAREPIZZA: must provide a count');
+    function mint(address toPizzaiolo, uint8 count) public virtual override onlyOwner {
```

Comment:
Can owner mint to any address? Should these not be assigned to multi sig wallet?

Reply:
owner can mint to any address yes